src/components/section/aside.jsx

```
@@ -0,0 +1,118 @@
+    };
+
+    const handleMouseEnter = (index) => {
+        setHoverIndex(index);
+    };
+
+    const handleMouseLeave = () => {
+        setHoverIndex(null);
+    };
+
+    return (
+        <header id='header' role='banner'>
```

Biome (2.1.2)

[error] Interactive elements should not be assigned non-interactive roles.

WAI-ARIA roles should not be used to convert an interactive element to a non-interactive element.
Wrap your interactive element in a <div> with the desired role or put the content inside your interactive element.
Unsafe fix: Remove the role attribute.


(lint/a11y/noInteractiveElementToNoninteractiveRole)